File: Java/extra/Lista 01/src/ex10.java
// Lê o raio de um círculo e mostre como saída o perímetro (2*π*Raio) e a área (π*Raio2).

import java.util.Scanner;
import java.math.*;

public class ex10 {
    static double perimetro(int a) {
        double x = 2 * Math.PI * (double) a;
        return x;
    }

    static double area(int a) {
        double y = Math.PI * Math.pow(a, 2);
        return y;
    }

    public static void main(String[] args) {
        int num;
        System.out.println("Digite o valor do raio: ");
        try (Scanner leitor = new Scanner(System.in)) {
            num = leitor.nextInt();
        }

        System.out.format("\nO perímetro do círculo é de %.2f cm\nA área do círculo é de %.2f cm²", perimetro(num),
                area(num));
    }
}

File: C/Listas/Lista 01/ex10.c
//Lê o raio de um círculo e mostre como saída o perímetro (2*π*Raio) e a área (π*Raio2).
#include <stdio.h>

int main(void){
    int raio;
    const float PI=3.14;
    printf("Informe o valor do raio do círculo: ");
    scanf("%d", &raio);

    printf("\nConforme o valor inserido, o perímetro do círculo equivale a %.1fm", 2*PI*raio);
    printf("\nA área do seu círculo será de %.1fm²", PI*(raio*raio));
}